test/rules/use_super_initializers_test.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:linter/src/rules/use_super_initializers.dart';
+import 'package:test/expect.dart';
+import 'package:test_reflective_loader/test_reflective_loader.dart';
+
+import '../rule_test_support.dart';
+
+main() {
+  defineReflectiveSuite(() {
+    defineReflectiveTests(UseSuperInitializersTest);
+  });
+}
+
+@reflectiveTest
+class UseSuperInitializersTest extends LintRuleTest {
+  @override
+  List<String> get experiments => [
+        EnableString.super_parameters,
+      ];
+
+  @override
+  String get lintRule => 'use_super_initializers';
+
+  test_message_plural_2() {
```

Comment:
This and the next method ought to already be covered by the `IterableOfStringExtensionTest`.

Reply:
yeah.  I just wanted to sanity check.  No need to keep them now.  Thanks.